src/discussions/posts/post/PostLink.jsx

```diff
@@ -95,6 +90,10 @@ function PostLink({
                   <span className="sr-only">{' '}reported</span>
                 </Badge>
               )}
+
+              {post.pinned && (
+                <Icon src={PushPin} className={postReported || showAnsweredBadge ? 'ml-2' : 'ml-auto'} />
+              )}
```

Comment:
We are using 20x20 icons in the post summary card. Please use the `icon-size` class which will set the icon height and width. Please make sure the height should not exceed 22px.